tests/unit/Codeception/Command/GenerateScenarioTest.php

```diff
@@ -37,29 +37,33 @@ protected function setUp()
     public function testBasic()
     {
         $this->execute(array('suite' => 'dummy'));
-        $this->assertEquals(codecept_root_dir().'tests/data/scenarios/dummy/File_Exists.txt', $this->filename);
-        $this->assertContains('I WANT TO CHECK CONFIG EXISTS', $this->content);
-        $this->assertContains('I see file found "$codeception"', $this->content);
+        $this->assertArrayHasKey($file = codecept_root_dir().'tests/data/scenarios/dummy/File_Exists.txt', $this->saved);
+        $content = $this->saved[$file];
+        $this->assertContains('I WANT TO CHECK CONFIG EXISTS', $content);
+        $this->assertContains('I see file found "$codeception"', $content);
         $this->assertContains('* File_Exists generated', $this->output);
     }
 
     public function testMultipleTestsGeneration()
     {
         $this->execute(['suite' => 'dummy']);
-        $fileNames = array_map(function ($f) { return $f['filename']; }, $this->log);
+        $fileNames = array_keys($this->saved);
```

Comment:
$fileNames could be eliminated by using assertArrayHasKey instead of assertContains